xla/service/gpu/runtime/command_buffer_cmd.h

```diff
@@ -111,35 +112,15 @@ std::string CommandBufferCmdString(CommandBufferCmdType type);
 //
 // Commands must be thread safe as they can be recorded into multiple command
 // buffers concurrently on different stream executors.
+using xla::BufferUse;
```

Comment:
using declaration inside headers are banned, also you are already in xla namespace so it's not needed ere

Reply:
Fixed.